index.bs

```diff
@@ -779,6 +780,11 @@ To <dfn>initialize WebTransport over HTTP</dfn>, given a {{WebTransport}} object
     1. Let |error| be a newly [=DOMException/created=] {{WebTransportError}} whose
        {{WebTransportErrorOptions/source}} is `"session"`.
     1. [=Cleanup=] |transport| with |error|.
+  1. If [=verify a certificate hash|verifying the certificate hash=] from |connection| against
```

Comment:
I'm not sure this is a good place to put it.  This is not a separate step, but rather a modification of `obtaining a connection` step (which is where cert check would normally happen).  The problem is, `obtaining a connection` does not actually accept an argument that would let you modify certificate verification, nor does it talk about it at all, so I'm afraid we would have to be somewhat vague about this.

Reply:
Do you think it makes sense to move this to the end of Step 7.2, or is there a better place you would suggest?

Reply:
Yes, something along the lines of "When obtaining a connection, if serverCertificatHashes is specified, instead of the default certificate version algorithm, validate the certificate against custom cert requirements and then validate it against the provided hashes".

Reply:
Got it, thank you!